freeipa.tf

```diff
@@ -50,7 +49,6 @@ module "ipa0" {
     data.terraform_remote_state.cdm.outputs.cdm_security_group.id,
   ]
   subnet_id = data.terraform_remote_state.networking.outputs.private_subnets[local.subnet_cidrs[0]].id
-  tags      = merge(var.tags, map("Name", "FreeIPA 0"))
```

Comment:
I removed the `Name` tag because it's fairly inconvenient in this case.  I'd have to do one of the following:
* Add an `additional_tags` variable to [cisagov/freeipa-server-tf-module](https://github.com/cisagov/freeipa-server-tf-module) and pass the `Name` tag in there.
* Create a separate provider for each instance of the [cisagov/freeipa-server-tf-module](https://github.com/cisagov/freeipa-server-tf-module) Terraform module, with the difference between the providers being only the value of the `Name` tag.

If we really want to keep the `Name` tag on the EC2 instances then I'd prefer the second option.  What do you think @dav3r?

Reply:
Upon further consideration I have decided that the `Name` tag is sometimes useful, so I have implemented the solution in the second bullet.  See commit ec56ff4.

Reply:
I have used this code to deploy a new AMI to our staging COOL environment, so I am (even more) confident that it functions as expected.

Reply:
Excellent! Thanks for circling back to mention that you'd deployed using this branch's code.